src/iframe/src/Iframe.js

```diff
@@ -478,6 +499,9 @@ class Iframe extends Component {
       // Otherwise, wait for messages from parent.
       window.addEventListener('message', handleMessage)
     }
+
+    this.renderer = params.get('renderer') || "framebuffer"
+    this.updateGlobals()
```

Comment:
Why do we need to call `this.updateGlobals()` here?

Reply:
See above